README.md

```diff
@@ -0,0 +1,74 @@
+# Cookiecutter PyPackage
+
```

Comment:
Badge for license

Reply:
Actually, yea I am going to convert this template to MIT License and put a badge. This template should be AICS licensed...